wolox-books/src/app/app-routing.module.ts

```diff
@@ -1,8 +1,13 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { LoginFormComponent } from './screens/auth/screens/login/components/login-form/login-form.component';
+import { SignupFormComponent } from './screens/auth/screens/signup/components/signup-form/signup-form.component';
 
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'login', component: LoginFormComponent },
+  { path: 'signup', component: SignupFormComponent }
```

Comment:
It would be a good idea to have a page not found component on case the route doesnt match any of this two routes

Reply:
it would also be a good idea to redirect the user to the login if the route is blank

Reply:
I'm going to fix this in a following PR since this structure changed a lot in different PR's